cmd/server_test.go

```diff
@@ -174,6 +179,7 @@ func TestExecute_Defaults(t *testing.T) {
 	strExceptions := map[string]string{
 		GHUserFlag:                       "user",
 		GHTokenFlag:                      "token",
+		GiteaBaseURLFlag:                 "https://gitea-base-url.com",
```

Comment:
Can we default this to `localhost` instead ? Seems a little risky to put a real domain name when we do not control it

```suggestion
		GiteaBaseURLFlag:                 "http://localhost",
```